### ISSUE TITLE
Can anyone send me a release?

### ISSUE THREAD

**Issue by chinabestcountry** [danielkrupinski/Osiris#3699]
i complie it but i didn't work!Can any one send me a release?Thx so muchhhh

**Comment by milkteaaa** [comment]
it's not hard to follow the instructions
where did you mess up?

**Comment by wilarN** [comment]
Try to do it yourself because the instructions are there and they're very clear, but if you still really need help add me on discord and i'll try my best to guide you. william_#6737

**Comment by chinabestcountry** [comment]
i build it by following the instructions and i inject to csgo.exe,but when i pressed my ins there wasn't any menu
![image](https://user-images.githubusercontent.com/94364764/173584634-d8c062b0-b140-4712-bab6-8ab8d68787cf.png)
can someone help me ? plzzzz and thx u so much

**Comment by chinabestcountry** [comment]
btw my discord is T__A__T#1370

**Comment by Patrixsin** [comment]
so its injector problem. Find better injector

**Comment by chinabestcountry** [comment]
i use  CSGhost injector to inject and i worked.but later osiris showed me this: 
![image](https://user-images.githubusercontent.com/94364764/174438437-034052af-6298-44b4-8c39-f41cc098a405.png)

**Comment by chinabestcountry** [comment]
thanks about Willam's help.The problem has been solve
